doc/pages/administrator/095-advanced-operations.md

```diff
@@ -134,3 +134,108 @@ After you end performing all the steps above, the interface will be completely r
 You can then proceed to install a new interface with the same name and major version without any
 conflict. *Remember to remove the interface also on the device side, otherwise devices will keep
 getting disconnected if they try to publish on the deleted interface.*
+
+## Handling Astarte when uninstalling the Operator
+
+Installing the Astarte Operator is as simple as installing its Helm chart. Even if the
+install and upgrade procedures are very simple and straightforward, the design choices behind the
+development of the Operator must be taken into account to avoid undesired effects while handling the
+Operator's lifecycle.
+
+The installation of the Operator's Helm chart is responsible for the creation of RBACs, the creation
+of the Operator's deployment and the installation of Astarte CRDs. The fact that all the CRDs
+installed with the Helm chart are templated has some important consequences: if on one hand this
+characteristic ensures great flexibility in configuring your Astarte instance, on the other hand it
+entails the possibility of deleting the CRDs by simply uninstalling the Operator.
+
+The following sections will highlight what happens under the hood while uninstalling the Operator
+and show the suggested path to restore your Astarte instance after the removal of the Operator.
+
+Please, read carefully the following sections before taking any actions on your cluster and be aware
+that improper operations may have catastrophic effects on your Astarte instance.
+
+### What happens when uninstalling the Operator
+
+The Operator's installation procedure marks all the Astarte CRDs as owned by the Operator itself.
+Therefore, when the Operator is uninstalled all the CRDs are seen as orphaned and the Kubernetes
+controller automatically set them as ready to be deleted. Thus, when the Operator is uninstalled you
+end up with the following situation:
+- Flow and AstarteVoyagerIngress CRDs are deleted, along with the custom resources depending on
+  them;
+- Astarte CRD is marked for deletion, but its removal is postponed until the moment in which the
+  Astarte finalizer is executed.
+
+### Backup your resources
```

Comment:
Backup you resources should be moved to top level, it is more general than uninstall itself.